samples/getting-started/qrng/Qrng.qs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+namespace Qrng {
+    open Microsoft.Quantum.Intrinsic;
+
+    operation QuantumRandomNumberGenerator() : Result {
```

Comment:
Since this is an operation, as per the style guide, I'd suggest naming it as a verb to help remind that operations do something (contrast functions, which _are_ something).
```suggestion
    operation SampleQuantumRandomNumberGenerator() : Result {
```

Reply:
Ok, we should change it as well in the original QRNG tutorial.